Resources/Prototypes/Entities/Structures/Piping/Atmospherics/special.yml

```diff
@@ -25,3 +25,32 @@
   - type: Tag
     tags:
       - SpreaderIgnore
+
+- type: entity
+  id: AtmosDeviceFanDirectional
+  name: directional fan
+  description: A thin fan, stopping atmospherics across it.
```

Comment:
This might leave engineers perhaps a bit confused as to what "stopping atmospherics" means. May I suggest something like "that stops the movement of gases across it"